docs/UserGuide.md

```diff
@@ -1,192 +1,85 @@
-
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
-
-   * **`clear`** : Deletes all contacts.
-
-   * **`exit`** : Exits the app.
-
-1. Refer to the [Features](#features) below for details of each command.
-
---------------------------------------------------------------------------------------------------------------------
-
-## Features
-
-<div markdown="block" class="alert alert-info">
-
-**:information_source: Notes about the command format:**<br>
-
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
-
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
-
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
-
-</div>
-
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
-
-### Adding a person: `add`
-
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+**● Quick start**</br>
+**● Features**
+* Adding a dog/owner/program to database: add
+* Deleting a dog/owner/program profile: delete
+* Show the list of dogs based on search: list
+* Enrol a dog to a program: enrol
+* Drop a dog from a program: drop
+* View daily schedule: schedule
+* Editing raw file
+* Application instructions: help
+* Exiting the application: exit
+
+**● FAQ**</br>
+**● Command Summary**
+
+###**Quick start**
+1. Ensure you have **Java 11 or above** installed in your Computer.
+2. Download the **latest** pawbook.jar.
+3. Copy the file to the folder you want to use as the home folder for your
+   Pawbook.
+4. Double-click the file to start the app.
+5. For new users, type help in the command box to view the instruction list
+6. Type the command in the command box and press Enter to execute it. e.g.
+   typing add and pressing Enter will allow you to start adding information to
+   the database.
+
+**#Please refer to the features below for details of each command.**
```

Comment:
For headings I think it is fine to leave out the bold if we are going to use headings. It is also necessary to leave a space in between the `#` and the word after for it to take effect.

Reply:
I just wanted to leave the hash-tag before the line.

Reply:
As in the hash-tag is a hash-tag not used as heading? Because throughout the document it seems like they are intended to be headings, if you preview the rendered markdown.

Reply:
Ok, I will remove it.